packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```diff
@@ -115,3 +126,22 @@
 		margin: 0;
 	}
 }
+
+.edit-site-navigation-panel__template-item-title {
+	display: inline-block;
+	padding-right: $grid-unit-20;
+	position: relative;
+}
+
+.edit-site-navigation-panel__template-item-customized-dot {
+	position: absolute;
+	right: 0;
+	top: 50%;
+	margin-top: -$grid-unit-05;
+	width: $grid-unit-10;
+	height: $grid-unit-10;
+	display: block;
+	background: var(--wp-admin-theme-color);
+	border-radius: 50%;
+	cursor: default;
```

Comment:
`default` doesn't seem very fitting since it doesn't really denote that something is happening.  While it needs to be different from `pointer` that is enabled in its surrounding element, perhaps something like `help` or `wait` would be more fitting.  The tooltip does take a second to pop up, so something other than `default` would be nice so it would convey to the user to expect 'something' from the interaction.

Reply:
`help` could be good!